src/components/WelcomeModal/WelcomeModal.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import ModalWithForm from "../ModalWithForm/ModalWithForm";
+
+const WelcomeModal = ({
+  handleCloseModal,
+  isOpen,
+  buttonText,
+  onSubmit,
+  onLogInModal,
+  onRegisterModalOnWelcomeModal,
+  isButtonDisabled,
+  isMicActivated,
+  isMicActivationPopupVisible,
+  onMicActivation,
+}) => {
+  return (
+    <div className="welcome__modal">
+      <ModalWithForm
+        buttonText={buttonText}
+        onClose={handleCloseModal}
+        isOpen={isOpen}
+        modalName={"Welcome_Modal"}
+        isButtonDisabled={isButtonDisabled}
```

Comment:
* Please, pass `isValid` into the props of `ModalWithForm` to disable the submit button if the form is **not** valid